Rakefile

```diff
@@ -1,28 +1,37 @@
 #!/usr/bin/env rake
-require 'bundler/gem_tasks'
-require 'rspec/core/rake_task'
-require 'net/http'
-require 'net/https'
+# frozen_string_literal: true
+require "bundler/gem_tasks"
+require "rspec/core/rake_task"
+require "net/http"
+require "net/https"
 
 desc "Run RSpec"
+# RSpec::Core::RakeTask.new(:spec)
```

Comment:
Fixed now. 

FYI, I removed the `desc "Run Rspec"` line as well as it overrode the description it inherits from RSpec.